addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -482,6 +483,28 @@ def _compute_delay(self):
             else:
                 applicant.delay_close = False
 
+    @api.depends('kanban_state', 'application_status', 'date_closed')
+    def _compute_rotting(self):
+        super()._compute_rotting()
+
+    @api.depends('stage_id.day_rot')
+    def _compute_date_rot(self):
+        return super()._compute_date_rot()
+
+    def _resource_is_not_rotting_hook(self, applicant):
+        if applicant.application_status != 'ongoing' or applicant.kanban_state == 'blocked' or applicant.date_closed:
```

Comment:
Was the "blocked" part discussed? I don't really see why "blocked" would mean it's not rotting, it could be blocked with no activity set on it and nobody paying attention to it, for example.